backend/de.metas.banking/de.metas.banking.base/src/main/java/de/metas/banking/payment/impl/PaySelectionBL.java

```diff
@@ -54,6 +59,14 @@ public class PaySelectionBL implements IPaySelectionBL
 	private static final AdMessageKey MSG_PaySelectionLines_No_BankAccount = AdMessageKey.of("C_PaySelection_PaySelectionLines_No_BankAccount");
 
 	private final IPaySelectionDAO paySelectionDAO = Services.get(IPaySelectionDAO.class);
+	private final IInvoiceBL invoiceBL = Services.get(IInvoiceBL.class);
+	private final IBPBankAccountDAO bpBankAccountDAO = Services.get(IBPBankAccountDAO.class);
+
+	@Override
+	public I_C_PaySelection getByIdOrError(@NonNull final PaySelectionId paySelectionId)
+	{
+		return Check.assumePresent(getById(paySelectionId), "Pay Selection should be present");
```

Comment:
why not simply calling 
return getById(paySelectionId).orElseThrow(()->new AdempiereException(....) 

?